cmd/snap/cmd_quota.go

```diff
@@ -144,6 +144,9 @@ func parseCpuQuota(cpuMax string) (count int, percentage int, err error) {
 		if err != nil || count == 0 {
 			return 0, 0, parseError(cpuMax)
 		}
+	} else {
+		// Assume format was M%
+		count = 1
```

Comment:
So does --cpu 5% mean 1 CPU, 5% or all CPUs 5%?

Reply:
--cpu 5% means 1 cpu 5%, --cpu2x10% means 1 cpu 20%, the maximum percentage you can specify is 100%, to use more than 100% you need to use the two part (2x100%)

Reply:
Thank you for this question, this just brought some shortcomings to light!